README.md

```diff
@@ -483,7 +485,7 @@ This would ensure that cross-site cookies are hostname bound and only sent over
 
 ### Memory impact
 
-Based on Chrome data, we estimate limiting a domain to 10 cookies per-partition will satisfy ~99% of existing cross-site cookie use cases on the web today.
+Based on Chrome data, we estimate that 10 cookies per-partition will satisfy ~99% of existing cross-site cookie use cases on the web today.
```

Comment:
Kind of unrelated, but we actually have browser metrics for this that show this was an underestimate on the number of cookies domains use. Wondering if this section needs a refactor.

Reply:
Ok, do you want to do that at some point? I don't really have a good overview of those metrics.